.readthedocs.yaml

```diff
@@ -1,8 +1,14 @@
 version: 2
 
+build:
+  os: "ubuntu-20.04"
+  tools:
+    python: "mambaforge-4.10"
+
+python:
+    install:
+       - method: setuptools
+         path: .
```

Comment:
I don't think this is needed, since `environment-docs.yml` already specifies installing xmovie.